### ISSUE TITLE
eclipse build error

### ISSUE THREAD

**Issue by GoogleCodeExporter** [BinaryBlue/imsdroid#210]
```
What steps will reproduce the problem?
1. checkout imsdroid in eclipse windows
2. try to build

What is the expected output? What do you see instead?
the expected output is a built version.

What version of the product are you using? On what operating system?
eclipse 3.6.2
adt 10
sdk android 1.6

Please provide any additional information below.
errors such as:


Description Resource    Path    Location    Type
android.provider.ContactsContract cannot be resolved to a 
variable    ContactService.java /Main/src/org/doubango/imsdroid/Services/Impl   line 
440 Java Problem
The method acquire() of type NetworkService must override a superclass 
method  NetworkService.java /Main/src/org/doubango/imsdroid/Services/Impl   line 
150 Java Problem


Thanks,
Ronny


```

Original issue reported on code.google.com by `Ronny.Ar...@gmail.com` on 5 Apr 2011 at 12:05

**Comment by GoogleCodeExporter** [comment]
```
You should build the source code using sdk 2.0 even if you are targeting 1.6 
because "android.provider.ContactsContract" is only available staring 2.0.
For all errors saying "XX must override a superclass method YY":
1. Open eclipse, right click on your project then select Properties
2. Select "Java Compiler" from the left and change the version from 1.5 to 1.6
3. Select OK
```

Original comment by `boss...@yahoo.fr` on 5 Apr 2011 at 7:03
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
Original comment by `boss...@yahoo.fr` on 13 Jul 2011 at 11:54
- Changed state: **Done**
